cli/on-boarding/terraform.md

```diff
@@ -5,73 +5,159 @@ description: Learn how to onboard Terramate to any existing Terraform project wi
 
 # Get started with Terramate and Terraform
 
-This page will teach you how to onboard Terramate to an existing Terraform project.
+This guide will teach you how to onboard Terramate to any existing Terraform repository in less than 5 minutes.
 
-Terramate can be added to any existing Terraform project with a single command
-and without changing any existing configuration, making the onboarding **seamless**
-and **fast**.
+Here's what you will learn:
+- Install Terramate CLI
+- Onboard Terramate CLI to any Terraform repository
+- Sign up for a free Terramate Cloud account
+- Sync your stacks to Terramate Cloud
 
-::: tip
-If instead, you'd like to learn how to get started with Terramate from scratch, take a look at our
-[Getting Started Guide](../getting-started/index.md).
+## 1: Install Terramate
+
+Install the Terramate cli:
+
+ ::: code-group
+```sh [macOS]
+brew install terramate
+```
+
+```sh [Ubuntu & Debian]
+# Add the Terramate repo to your sources
+echo "deb [trusted=yes] https://repo.terramate.io/apt/ /" \
+  | sudo tee /etc/apt/sources.list.d/terramate.list
+
+apt update
+apt install terramate
+```
+
+```sh [Fedora & CentOS]
+# Add the Terramate repo to your sources
+sudo tee /etc/yum.repos.d/terramate.repo <<EOF
+[terramate]
+name=Terramate Repository
+baseurl=https://repo.terramate.io/yum/
+enabled=1
+gpgcheck=0
+EOF
+
+dnf install terramate
+```
+
+```txt [Windows]
+Download the binary from
+https://github.com/terramate-io/terramate/releases
+```
 :::
 
-<!-- The following sections will teach you how to:
-- Import Terraform root modules as Terramate stacks
-- Orchestrate  -->
+For other installation methods, please see the [installation page](../installation.md).
 
-## Import Existing Terraform Stacks
+## 2: Import Terraform root modules to Terramate
 
-To onboard Terramate to an existing Terraform project, you first need to import your Terraform root modules into Terramate.
+Mark your Terraform root modules as Terramate stacks:
 
 ```bash
 terramate create --all-terraform
 ```
+This command creates a `stack.tm.hcl` file in every existing Terraform root module, enabling Terramate to consider the Terraform root modules as [stacks](../stacks/index.md). 
 
-This command will detect existing Terraform root modules and create a stack configuration in them, which Terramate
-requires to identify a Terraform root module as a stack.
+For example, an existing Terraform project repo with two root modules:
+```bash
+.
+├── alice
+│   ├── main.tf
+│   └── terraform.tfstate
+├── bob
+│   ├── main.tf
+│   └── terraform.tfstate
```

Comment:
```suggestion
├── alice
│   ├── main.tf
│   └── backend.tf
├── bob
│   ├── main.tf
│   └── backend.tf
```

Reply:
Technically, the `terramate create --all-terraform` command doesn't look for a `terraform.tfstate` state file but for a state backend configuration in any file within a directory which is configured using the `backend {}` block